stable/nginx-ingress/templates/controller-cluster-role.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.controller.rbac.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "fullname" . }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - endpoints
+      - nodes
+      - pods
+      - secrets
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
```

Comment:
when using `--watch-namespace` I had to add `namespaces` here to avoid the following error:

```bash
F0710 18:20:42.008164      10 launch.go:145] no watchNamespace with name stage-1-osf found: User "system:serviceaccount:stage-1-osf:mfjzdy-nginx-ingress" cannot get namespaces in the namespace "stage-1-osf".: "Unknown user \"system:serviceaccount:stage-1-osf:mfjzdy-nginx-ingress\"" (get namespaces stage-1-osf)
```